tools/cli/src/errors.rs

```diff
@@ -45,4 +45,10 @@ pub enum CLIError {
     /// Various errors related to I/O operations.
     #[error("{0:?}")]
     IOError(#[from] std::io::Error),
+
+    #[error("{0}")]
+    ManifestParseError(String),
```

Comment:
I suggest to introduce a new enum for this error variant, it's better to avoid using just `String` in such cases, though there are several cases when they are used and they should be fixed eventually.